src/test/java/com/google/sps/RankTest.java

```diff
@@ -37,81 +37,202 @@
-    return Arrays.asList(mon, wed, fri);
-  }
-
-  /**
-   * Returns a list of times on monday and friday, to be used in section constructors.
-   */
-  public static List<TimeRange> monFri(int hour, int minute, int durationMinutes) {
-    TimeRange mon = TimeRange.fromStartDuration(TimeRange.MONDAY, hour, minute, durationMinutes);
-    TimeRange fri = TimeRange.fromStartDuration(TimeRange.FRIDAY, hour, minute, durationMinutes);
-    return Arrays.asList(mon, fri);
-  }
-
-  /**
-   * Returns a list of times on Tuesday and Thursday, to be used in section constructors.
-   */
-  public static List<TimeRange> tuesThurs(int hour, int minute, int durationMinutes) {
-    TimeRange tues = TimeRange.fromStartDuration(TimeRange.TUESDAY, hour, minute, durationMinutes);
-    TimeRange thurs = TimeRange.fromStartDuration(TimeRange.THURSDAY, hour, minute, durationMinutes);
-    return Arrays.asList(tues, thurs);
+  
+  private static List<TimeRange> createSectionTimes(int[] days, int hour, int minute, int durationMinutes) {
+    List<TimeRange> timeRanges = new ArrayList<>();
+    for (int day : days) {
+      TimeRange timeRange = TimeRange.fromStartDuration(day, hour, minute, durationMinutes);
+      timeRanges.add(timeRange);
+    }
+    return timeRanges;
   }
 
   /**
    * Returns a list of courses in the order of priority that they should be considered
    */
   public static List<Course> createCoursesPriority() {
-    Section section1 = new Section("Professor A", monWedFri(9, 00, DURATION_90_MINUTES));
+    Section section1 = new Section("Professor A", createSectionTimes(MON_WED_FRI, 9, 00, DURATION_90_MINUTES));
     Course course1 = new Course("Operating System", "15410", 
         "Computer Science", 15, true, Arrays.asList(section1), Arrays.asList());
     
-    Section section2 = new Section("Professor B", tuesThurs(10, 30, DURATION_90_MINUTES));
+    Section section2 = new Section("Professor B", createSectionTimes(TUES_THURS ,10, 30, DURATION_90_MINUTES));
     Course course2 = new Course("Compilers", "15411", 
         "Computer Science", 15, true, Arrays.asList(section2), Arrays.asList());
 
-    Section section3 = new Section("Professor C", tuesThurs(12, 00, DURATION_90_MINUTES));
+    Section section3 = new Section("Professor C", createSectionTimes(TUES_THURS, 12, 00, DURATION_90_MINUTES));
     Course course3 = new Course("Algorithms", "15210", 
         "Computer Science", 15, true, Arrays.asList(section3), Arrays.asList());
 
-    Section section4 = new Section("Professor D", tuesThurs(13, 30, DURATION_90_MINUTES));
+    Section section4 = new Section("Professor D", createSectionTimes(TUES_THURS, 13, 30, DURATION_90_MINUTES));
     Course course4 = new Course("Experimental Physics", "33104", 
         "Physics", 33, true, Arrays.asList(section4), Arrays.asList());
 
     return Arrays.asList(course1, course2, course3, course4);
   }
 
+  public static List<Course> createCoursesPriorityBigTest() {
```

Comment:
This function and the one below it look really similar to the one in SchedulerTest. Since there seem to be some shared functionality, you could probably pull out some of this course/schedule creation to a separate file. No need to make the changes, just a comment.

Reply:
That makes sense, thanks for the tip.